### PR TITLE
Create the Quickstart Tutorial Notebook and rename some notebooks

### PR DESCRIPTION
**Pull request by kenxu95**
## Describe your changes and why you are making these changes
- Moved all the tutorial notebooks under tutorials/
- Added a name for the wine prediction notebook
- Created a short Quickstart Tutorial that just goes over the basics of the flow on the frontpage of the repo

The gitbook side of this change: https://github.com/aqueducthq/gitbook/pull/32

There's a bunch of stuff I noticed that could be improved, but was out of scope of this task. Noted those here: https://linear.app/aqueducthq/issue/ENG-1820/make-improvements-to-our-internal-example-notebooks

## Related issue number (if any)
ENG-1546

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: examples/tutorials/Quickstart Tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,352 @@
+{
```
Comment (kenxu95) [inline examples/tutorials/Quickstart Tutorial.ipynb:1]: I just saw that we actually have another Quickstart guide here: https://docs.aqueducthq.com/quickstart-guide, which is slightly different from (but very similar to) this notebook, which mirrors the example on the frontpage of our repo.

We should probably only have one of these quickstarts right? But I'm not sure how to merge them or which one to pick... The one on the website is written more like a guide and this one is written more like a quick example to get up and running Also, the one on the website talks about metrics, whereas this one doesn't. cc: @vsreekanti

Comment (vsreekanti) [reply examples/tutorials/Quickstart Tutorial.ipynb:1]: Sorry for the confusion. Anecdotally, the quickstart guide has been working pretty well for people who've jumped on calls with us, so feel free to update it with anything you feel is helpful from this, but I'd like to keep most of what's currently there if that's cool with you @kenxu95!

Comment (kenxu95) [reply examples/tutorials/Quickstart Tutorial.ipynb:1]: ok sg - so I should translate the quickstart guide into a notebook then, and stick that in the "Example Workflows" section? as the "Quickstart Tutorial"

Comment (vsreekanti) [reply examples/tutorials/Quickstart Tutorial.ipynb:1]: yep!

**Review by cw75 — APPROVED** [review]
@kenxu95 the content has already been reviewed on the Gitbook side right? If so we can just stamp this.

**Pull request merged**